content/6_Advanced/SCC.mdx

```diff
@@ -99,6 +99,76 @@ int main() {
 
 </CPPSection>
 
+
+<JavaSection>
+
+```java
+
+import java.io.*;
+import java.util.*;
+
+public class Main {
+	static boolean[] vis;
+	static List<Integer>[] adj;    // // Adjacency list of neighbor
+	static List<Integer>[] adj_t;  // adj_t is the transpose of adj
+	static int[] id;
+	static List<Integer> S;
+	public static void main(String[] args) {
+		Kattio io = new Kattio();
+		int n = io.nextInt();
+		int m = io.nextInt();
+		adj = new ArrayList[n + 1];
+		adj_t = new ArrayList[n + 1];
+		vis = new boolean[n + 1];
+		id = new int[n + 1];
+		S = new ArrayList<>();
```

Comment:
initialize variables above where you actually use them

Reply:
Fix it